pylmod/base.py

```diff
@@ -80,7 +90,20 @@ def _url_format(self, service):
         return base_service_url
 
     def rest_action(self, func, url, **kwargs):
-        """Routine to do low-level REST operation, with retry"""
+        """Routine to do low-level REST operation, with retry.
+
+        Args:
+            func (callable): API function to call
+            url (str): service URL endpoint
+            kwargs (dict): addition parameters
+
+        Raises:
+            requests.RequestException
+            ValueError
+
+        Returns:
+            json
```

Comment:
I'm not sure the best way to say this, but this doesn't return json, it returns json that has been converted into a python data structure, which is an important distinction, because if you ran `json.loads(blah.rest_action(...)` you get an exception ( TypeError: expected string or buffer), so maybe something `list or dict` would make more sense.